pyrealm/core/solar.py

```diff
@@ -670,14 +670,9 @@ def calc_solar_elevation(site_obs_data: LocationDateTime) -> NDArray:
     altitude on perceived solar elevation.
 
     Args:
-        site_obs_data   : A data object containing the location and
-                            time-specific information for the observation site,
-                            including
-            latitude_rad: site latitude in radians
-            longitude   : site longitude in decimal degrees
-            UTC offset  : Offset from UTC for site local meridian, decimal degrees
-            local time  : decimal local time of observations at site,
-            Julian days : Julian day of observations at site
+        site_obs_data: A :class:`~pyrealm.core.calendar.LocationDateTime` instance
+            containing the location and time-specific information for the observation
+            site.
```

Comment:
This avoids repetition of the class structure by pointing to the specific docs (but also, the resulting rendering of this RST structure was all kinds of nasty).